src/main/java/net/snowflake/ingest/streaming/OpenChannelRequest.java

```diff
@@ -16,6 +17,11 @@ public enum OnErrorOption {
     // indexes. No data is ingested
   }
 
+  public enum ChannelType {
+    CLOUD_STORAGE,
+    ROWSET_API,
+  }
```

Comment:
I think this exists already? If you look at `StreamingResource` there's a check somewhere that compares against an enum.

Reply:
Yes, but the one used before was hided in the SDK, now this is a public enum in the public class